### PR TITLE
Remove useless == true

### PR DESCRIPTION
**Pull request by teoli2003**
Most of the time `if (myVar === true)` can be replaced by the simpler `if (myVar)`. This PR fixes this.

### CODE REVIEW

File: files/en-us/web/api/htmlelement/dataset/index.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ delete el.dataset.dateOfBirth;
 // Result on JS: el.dataset.dateOfBirth === undefined
 // Result on HTML: <div id="user" data-id="1234567890" data-user="johndoe">John Doe</div>
 
-if ('someDataAttr' in el.dataset === false) {
+if (!'someDataAttr' in el.dataset) {
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlelement/dataset/index.md:117]:
```suggestion
if (!('someDataAttr' in el.dataset)) {
```

File: files/en-us/web/api/gamepadbutton/pressed/index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ the button is currently pressed (`true`) or unpressed (`false`).
 ```js
 let gp = navigator.getGamepads()[0]; // Get the first gamepad object
 
-if(gp.buttons[0].pressed == true) {
+if(gp.buttons[0].pressed) {
```
Comment (Josh-Cena) [inline files/en-us/web/api/gamepadbutton/pressed/index.md:24]:
```suggestion
if (gp.buttons[0].pressed) {
```

File: files/en-us/web/api/gamepadbutton/touched/index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ A {{jsxref("Boolean")}}. True if touched.
 ```js
 let gp = navigator.getGamepads()[0]; // Get the first gamepad object
 
-if(gp.buttons[0].touched == true) {
+if(gp.buttons[0].touched) {
```
Comment (Josh-Cena) [inline files/en-us/web/api/gamepadbutton/touched/index.md:30]:
```suggestion
if (gp.buttons[0].touched) {
```

File: files/en-us/web/api/window/location/index.md [hunk 4]
```diff
@@ -192,7 +192,7 @@ const showBookmark = (function () {
   _scrollY = document.documentElement.scrollTop;
   _scrollX = document.documentElement.scrollLeft;
   _bookMark = sBookmark;
-  _useHash = bUseHash === true;
+  _useHash = bUseHash;
```
Comment (Josh-Cena) [inline files/en-us/web/api/window/location/index.md:195]: I'm not totally sure what this is doing, just want to point out you may still want to keep the original so `_useHash` is always a boolean

File: files/en-us/web/javascript/reference/errors/missing_parenthesis_after_condition/index.md [hunk 5]
```diff
@@ -78,12 +78,19 @@ if (done is true) {
 Instead you need to use a correct [comparison operator](/en-US/docs/Web/JavaScript/Reference/Operators).
 For example:
 
-```js example-good
+```js
 if (done === true) {
  console.log("we are done!");
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/errors/missing_parenthesis_after_condition/index.md:83]:
```suggestion
  console.log("we are done!");
```

File: files/en-us/web/javascript/reference/errors/missing_parenthesis_after_condition/index.md [hunk 6]
```diff
@@ -78,12 +78,19 @@ if (done is true) {
...
 }
 ```
 
+Or even better:
+```js example-good
+if (done) {
+ console.log("we are done!");
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/errors/missing_parenthesis_after_condition/index.md:90]:
```suggestion
Or even better:

```js example-good
if (done) {
  console.log("we are done!");
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/reference/errors/missing_parenthesis_after_condition/index.md (hunk 6 above)
Comment (teoli2003) [reply files/en-us/web/javascript/reference/errors/missing_parenthesis_after_condition/index.md]: Fixed

File: files/en-us/web/api/window/location/index.md (hunk 4 above)
Comment (teoli2003) [reply files/en-us/web/api/window/location/index.md:195]: I'm not sure about the whole example, so let's keep it.

Comment (teoli2003) [inline files/en-us/web/api/window/location/index.md:195]:
```suggestion
  _useHash = bUseHash === true;
```

File: files/en-us/web/api/webgl_api/by_example/color_masking/index.md [hunk 7]
```diff
@@ -102,7 +102,7 @@ window.addEventListener("load", function setupAnimation (evt) {
       || evt.target === bluetoggle && 2
       || 0;
     mask[index] = !mask[index];
-    if (mask[index] === true)
+    if (mask[index])
```
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/api/webgl_api/by_example/color_masking/index.md]: DOne.

File: files/en-us/web/api/webgl_api/by_example/color_masking/index.md [hunk 8]
```diff
@@ -102,11 +102,7 @@ window.addEventListener("load", function setupAnimation (evt) {
       || evt.target === bluetoggle && 2
       || 0;
     mask[index] = !mask[index];
-    if (mask[index] === true)
-      evt.target.textContent ="On";
-    else
-      evt.target.textContent ="Off";
-    gl.colorMask(mask[0], mask[1], mask[2], true);
```
Comment (Josh-Cena) [inline files/en-us/web/api/webgl_api/by_example/color_masking/index.md:109]: This should be kept?

Comment (teoli2003) [reply files/en-us/web/api/webgl_api/by_example/color_masking/index.md:109]: Mmmh, yes, I think I selected to quickly.

Comment (teoli2003) [reply files/en-us/web/api/webgl_api/by_example/color_masking/index.md:109]: Fixed

**Review by Josh-Cena — APPROVED** [review]
LGTM

**Pull request merged**